package.json

```diff
@@ -30,6 +30,7 @@
     "test:emulator": "bash ./scripts/emulator-tests/run.sh",
     "test:extensions-deploy": "bash ./scripts/extensions-deploy-tests/run.sh",
     "test:extensions-emulator": "bash ./scripts/extensions-emulator-tests/run.sh",
+    "test:frameworks": "bash ./scripts/frameworks-tests/run.sh",
```

Comment:
Some test is better than no test, but I think for this test to really mean something we need to test project initialization too. We can add that later though.